### DISCUSSION TITLE
Is the order of submission of the jobs stored somewhere in the Batch?

### DISCUSSION THREAD

**Discussion by HGSilveri** [pasqal-io/pasqal-cloud#91] [Q&A] [answered]
Hey,

I just realized that when retrieving the jobs in `Batch.jobs`, they are not necessarily in the same order as they were given in `SDK.create_batch()`. 
I was wondering if this order is at least stored somewhere so I don't have to go matching job IDs to jobs to sort the job results in order of submission.

Thanks in advance!

**Comment by MatthieuMoreau0** [comment, marked as answer]
Hey, yes as you pointed out jobs are not returned in order unfortunately which was a side effect of us moving from interger IDs to UUIDs. We have a ticket in our backlog to make sure jobs are returned in the same order they were created. This should be done in the coming weeks.

In the meantime, one workaround would be to match the job variables in the response with the ones you used when creating your batch. Sorry for the inconvenience.

**Reply by HGSilveri** [reply]
That's okay, I just wanted to confirm I wasn't missing something. Thanks!
